qutebrowser/completion/completer.py

```diff
@@ -197,13 +197,19 @@ def schedule_completion_update(self):
         For performance reasons we don't want to block here, instead we do this
         in the background.
         """
-        if (self._cmd.cursorPosition() == self._last_cursor_pos and
+        cmd = self._cmd.text().split(' ', 1)
+        if (len(cmd) > 1 and
+                0 < len(cmd[1]) < config.val.completion.min_chars and
+                self._cmd.cursorPosition() > self._last_cursor_pos):
```

Comment:
Is this so updates when moving the cursor to the left are immediate? (might want to explain this in the docstring or so)

Reply:
yes, exactly. I'll add docs